interview-questions.md

```diff
@@ -8,36 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
+Your answer: We could use add_foreign_key and add it to the students migration file. We then need to be sure to pass the changes using rails db:migrate
 
-Researched answer:
+Researched answer: We need to first generate a migration to add the cohort_id:integer. Then we would add_foreign_key.
 
 2. Which RESTful routes must always be passed params? Why?
 
-Your answer:
+Your answer: Put and delete must always be passed the ID param to declare which object will be updated or deleted.
 
-Researched answer:
+Researched answer: Depending on its usage, get may also require params, such as show or edit.
```

Comment:
The RESTful routes can be a little tricky, I know. Show, edit, update, destroy must be passed params